notebooks/200_models/201_fastflow.ipynb

```diff
@@ -0,0 +1,782 @@
+{
```

Comment:
<p>"we demonstrate how to initialize benchmark and custom datasets" reads a bit weird. Maybe "initialize benchmark- and custom datasets"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/386/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>